fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -86,9 +86,10 @@ auto_detect_jar_file() {
   local old_dir="$(pwd)"
   cd ${dir}
   if [ $? = 0 ]; then
-    local nr_jars="$(ls 2>/dev/null | grep -e '.*\.jar$' | grep -v '^original-' | wc -l | awk '{print $1}')"
+    # NB: Find both (single) JAR *or* WAR <https://github.com/fabric8io-images/run-java-sh/issues/79>
+    local nr_jars="$(ls 2>/dev/null | grep -e '.*\.jar$' -e '.*\.war$' | grep -v '^original-' | wc -l | awk '{print $1}')"
     if [ "${nr_jars}" = 1 ]; then
-      ls *.jar | grep -v '^original-'
+      ls 2>/dev/null *.jar *.war | grep -v '^original-'
```

Comment:
@rhuss this was wrong and breaks run-java.sh, I think because ls returns -1 if there is no ".jar or *.war, so re-opening #79 ...